ods_ci/tests/Tests/0200__rhoai_upgrade/0201__pre_upgrade.robot

```diff
@@ -67,16 +68,18 @@ Setting Pod Toleration Via UI
 Verify RHODS Accept Multiple Admin Groups And CRD Gets Updates
     [Documentation]    Verify that users can set multiple admin groups and
     ...    check OdhDashboardConfig CRD gets updated according to Admin UI
-    [Tags]      Upgrade     RHOAIENG-14306    Platform
+    [Tags]      Upgrade     RHOAIENG-14306    Platform      RHOAIENG-19806
     [Setup]     Begin Web Test
     # robocop: disable
     Launch Dashboard And Check User Management Option Is Available For The User
     ...    ${TEST_USER.USERNAME}
     ...    ${TEST_USER.PASSWORD}
     ...    ${TEST_USER.AUTH_TYPE}
     Clear User Management Settings
-    Add OpenShift Groups To Data Science Administrators     rhods-admins        rhods-users
-    Add OpenShift Groups To Data Science User Groups        system:authenticated
+    Set Global Variable         @{adm_groups}     rhods-admins        rhods-users
+    Set Global Variable         @{allwd_groups}     system:authenticated
```

Comment:
this won't work because post upgrade is executed after pre-upgrade in a separate process